core/src/main/java/io/neow3j/protocol/core/methods/response/StackItem.java

```diff
@@ -1,32 +1,38 @@
 package io.neow3j.protocol.core.methods.response;
 
-import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.DeserializationContext;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
-import io.neow3j.crypto.Base64;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.As;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
 import io.neow3j.model.types.StackItemType;
-import io.neow3j.protocol.core.methods.response.StackItem.StackDeserializer;
-import java.io.IOException;
-import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-
-@JsonDeserialize(using = StackDeserializer.class)
-public class StackItem {
 
+@JsonTypeInfo(use = Id.NAME, property = "type", include = As.EXISTING_PROPERTY)
+@JsonSubTypes(value = {
+        @JsonSubTypes.Type(value = AnyStackItem.class, name = StackItemType.ANY_VALUE),
+//        @JsonSubTypes.Type(value = PointerStackItem.class, name = StackItemType.POINTER_VALUE),
```

Comment:
Could you add a Todo comment for this? Then we could easily find what's missing in the future.